Compiler.coffee

```diff
@@ -70,8 +70,12 @@ class Compiler
         props.className = className
 
       React.cloneElement child, props,
-        React.Children.map child.props.children, (child) =>
-          @traverseChild child, parentClass
+        if React.Children.count(child.props.children) is 1
```

Comment:
Yeah it will work in the case if the one child is not already in an array. But there can be cases when it is in an array  (e.g. it can come from _.map where the mapped array has only one item - like a single email campaign).